Website/src/items/content/EdgeWebsite/Data/CMP Data/Testimonies/Striva.yml

```diff
@@ -23,6 +23,9 @@ Languages:
     - ID: "30de01d7-233e-42ac-bb14-751e9d60b1cd"
       Hint: Provider
       Value: Striva
+    - ID: "32fcc3c3-e6bb-4376-8bd8-bddb62fedb57"
+      Hint: LogoTransformation
+      Value: profile
```

Comment:
This should not be needed as it is already specified in the standard values. The whole block can be removed:
```
    - ID: "32fcc3c3-e6bb-4376-8bd8-bddb62fedb57"
      Hint: LogoTransformation
      Value: profile
```